lib/commands/pack.js

```diff
@@ -21,6 +21,11 @@ class Pack extends BaseCommand {
   static workspaces = true
   static ignoreImplicitWorkspace = false
 
+  constructor (...args) {
+    super(...args)
+    this.npm.config.set('foreground-scripts', true, 'default')
```

Comment:
This doesn't allow folks to explicitly disable foreground scripts.

Reply:
Or if it does, it's an approach we've never used before we've historically checked for if the user has overridden the default and then manually set it..  In  either case we need a test showing that folks can still override it.

Reply:
the `save` config is different in `npm update` and you can see how we've done it there


```js
     const save = this.npm.config.isDefault('save')
         ? false
         : this.npm.config.get('save')
```

This is an interesting approach, setting a default for the command, and may even be the right one.  We need to think through documentation on this too.

Reply:
I found it by looking at other commands, but I’m happy to change it to the pattern you’re describing.

Reply:
If we already do this elsewhere i'm less concerned about a new precedent.  Where did you find it?

Reply:
ahhh actually i found some *get*s with `default`, and then looked at Config see that it accepted a "where" of confTypes, and then just assumed it worked with `set` (which it may not, currently).

I'll make the change.